### PR TITLE
Add mg_toolkit_original_metadata

### PR DESCRIPTION
**Pull request by RZ9082**

### CODE REVIEW

File: tools_iuc.yaml [hunk 1]
```diff
@@ -2405,6 +2405,10 @@ tools:
     owner: iuc
     tool_panel_section_label: Metagenomic Analysis
 
+  - name: mg_toolkit_original_metadata
```
Comment (bgruening) [inline tools_iuc.yaml:2408]:
```suggestion
  - name: mg_toolkit
```

**Comment by bgruening** [comment]
@RZ9082 does this look corect?

**Comment by RZ9082** [comment]
> @RZ9082 does this look corect?

This is the folder yes. But i thought it has to be the tool id.

**Comment by bgruening** [comment]
Strictly is has to be the ToolShed repository name: https://github.com/galaxyproject/tools-iuc/blob/main/tools/mg_toolkit/.shed.yml#L1

Unfortunatately, you have this as the repo name here:
https://github.com/galaxyproject/tools-iuc/blob/main/tools/mg_toolkit/.shed.yml#L1

Which is not the same as the tool-ID

File: tools_iuc.yaml [hunk 2]
```diff
@@ -2405,6 +2405,10 @@ tools:
     owner: iuc
     tool_panel_section_label: Metagenomic Analysis
 
+  - name: mg_toolkit
```
Comment (bgruening) [inline tools_iuc.yaml:2408]:
```suggestion
  - name: mg_toolkit_original_metadata
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**